inst/modeldb/specificDrugs/Cirincione_2017_exenatide.R

```diff
@@ -0,0 +1,72 @@
+  reference <- "Cirincione B, Mager DE. Population pharmacokinetics of exenatide. British Journal of Clinical Pharmacology. 2017;83(3):517-526. doi:10.1111/bcp.13135"
+  covariateData <-
+    list(
+      AMT = "Dose (ug)",
+      DV = "Exenatide plasma concentration (pg/mL)",
+      eGFR = "Modification of Diet in Renal Disease estimate of glomerular filtration rate (mL/min/(1.73m^2))",
+      WT = "Baseline body weight (kg)",
+      STUDY1 = "Was the subject in study 1?  0 = no, 1 = yes (typically 0)",
+      STUDY5 = "Was the subject in study 5?  0 = no, 1 = yes (typically 0)"
+    )
+  # parameters are from Table 2 in the paper
+  ini({
+    lcl <- log(4.58) ; label("Linear clearance rate (L/hr)")
+    etalcl ~ log(1.339)
+    e_cl_gfr <- 0.838; label("Effect of eGFR on clearance (unitless)")
+    
+    lq <- log(3.72); label("Intercompartmental clearance (L/hr)") # written as Cld in the model table
+    
+    lkm <- log(567); label("Michaelis-Menten constant for clearance (pg/mL)")
+    etalkm ~ log(1.957)
+    
+    lvmax <- log(1.55); label("Maximum Michaelis-Menten clearance (ug/hr)")
+    
+    lvp <- log(7.04); label("Peripheral compartment volume (L)")
+    
+    lvc <- log(7.03); label("Typical central compartment clearance (L)")
+    etalvc ~ log(1.805)
+    e_vc_wt <- 2.67; label("Effect of body weight on central volume (unitless)")
+    
+    lkamax <- log(0.0813); label("Maximum first-order absorption rate (1/hr)")
+    lkmka <- log(16.9); label("Michaelis-Menten constant for absorption (ug)")
+    tau <- fixed(1.35); label("Duration of zero-order absorption")
+    fdepot <- fixed(1); label("Bioavailability (fraction)")
+    logitfr <- logit(0.628); label("Fraction of dose with first-order absorption")
+
+    expSdOther <- 0.373 ; label("Exponential residual error for all other studies")
+    expSdStudy1 <- 0.39 ; label("Exponential residual error for Study 1")
+    expSdStudy5 <- 0.08 ; label("Exponential residual error for Study 5")
+  })
+  model({
+    # cl equation is from table 2 in the paper
+    cl <- exp(lcl + etalcl)*(eGFR/80)^e_cl_gfr
+    q <- exp(lq)
+    km <- exp(lkm + etalkm)
+    vmax <- exp(lvmax)
+    vp <- exp(lvp)
+    # vc equation is from table 2 in the paper
+    vc  <- exp(lvc + etalvc)*(WT/84.8)^e_vc_wt
+    kamax <- exp(lkamax)
+    kmka <- exp(lkmka)
+    fr <- expit(logitfr)
+
+    kel <- cl/vc + vmax/(km*vc + central)
+    
+    # FIXME: Need to turn k0 off at time > tau
+    kzero <- (1-fr)*Dose/tau
+    # FIXME: Need to turn ka on at time > tau
+    ka <- fr*kamax/(kmka + depot)
+    d/dt(depot) <-  -ka*depot - kzero
+    d/dt(central) <- ka*depot + kzero - kel*central - k12*central + k21*peripheral1
+    d/dt(peripheral1) <-                              k12*central - k21*peripheral1
+    f(depot) <- fdepot
+    
+    cp <- central/vc
+    
+    OTHERSTUDY <- 1 - (STUDY1 + STUDY5)
+    propSd <- expSdOther*OTHERSTUDY + expSdStudy1*STUDY1 + expSdStudy5*STUDY5
+    cp ~ exp(propSd)
```

Comment:
@mattfidler, The residual error parameters for this model (`expSdOther`, `expSdStudy1`, and `expSdStudy5`) are not detected as residual error parameters by `mod$predDf$cond` here: https://github.com/nlmixr2/nlmixr2lib/blob/8a12b331ff267f3fdb40ef0d93bdb3b19acdfa81/R/modeldb.R#L129

Is this the correct way to code residual error that is conditional on study number?

Reply:
No.  You need a separate cmt or dvid for each endpoint.  Then you would code it as:

```R
cp.stud1 ~ prop(prop.sd.stud1)
cp.stud1 ~ prop(prop.sd.stud2)
...
```

Without this saem would not work.  Focei would estimate though, I think. However, some testing needs to be done here to make sure that the composite estimate is reasonable.

Reply:
I looked up the paper and made my modifications.  Note that these seem to be lognormally distributed.  One caution here (that probably should be mentioned), `nlmixr2` will require the concentrations to be on the **normal** scale not the log-transformed scale that is so commonly used in NONMEM.

Also this needs dvid/cmt definitions

Reply:
```
 ── Multiple Endpoint Model ($multipleEndpoint): ──  
  variable                       cmt                       dvid*
1   cp ~ … cmt='otherStudy' or cmt=4 dvid='otherStudy' or dvid=1
2   cp ~ …     cmt='study1' or cmt=5     dvid='study1' or dvid=2
3   cp ~ …     cmt='study5' or cmt=6     dvid='study5' or dvid=3
  * If dvids are outside this range, all dvids are re-numered sequentially, ie 1,7, 10 becomes 1,2,3 etc
```

Reply:
Thanks!